.gitignore

```diff
@@ -0,0 +1 @@
+
```

Comment:
- [ ] As requested by the previous reviewer, it would be nice if you added the node_modules directory to the .gitignore because it can be added by accident while setting up linters.

- [ ] [OPTIONAL] If I refer to the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md), you were required to create a test.md file that was to be ignored in the .gitignore. I think it would be great if you implement this feature of the project because from it we learn not to commit files that are not relevant to the project